packages/asunder.rb

```diff
@@ -1,41 +1,31 @@
-# Adapted from Arch Linux asunder PKGBUILD at:
-# https://github.com/archlinux/svntogit-community/raw/packages/asunder/trunk/PKGBUILD
+require 'buildsystems/autotools'
 
-require 'package'
-
-class Asunder < Package
+class Asunder < Autotools
   description 'a graphical CD ripper and encoder'
   homepage 'http://littlesvr.ca/asunder/'
-  version '2.9.7'
-  license 'GPL2'
+  version '3.0.1'
+  license 'GPL-2'
   compatibility 'x86_64 aarch64 armv7l'
-  source_url 'https://littlesvr.ca/asunder/releases/asunder-2.9.7.tar.bz2'
-  source_sha256 'c1c97cd34c04b8595e95df8a9a7dbc64a1e61f494b7a0cd2873802ad111874f4'
+  source_url "http://littlesvr.ca/asunder/releases/asunder-#{version}.tar.bz2"
+  source_sha256 '8868e2e1b97b6687c800e7f612262a316bb857edd39883768ce628b6d253376b'
   binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: '4c004dfe3a8287cdb7f422bd808f7cead56bfaa496ec915e5dcb0f2816dad742',
-     armv7l: '4c004dfe3a8287cdb7f422bd808f7cead56bfaa496ec915e5dcb0f2816dad742',
-     x86_64: '46865361e0deeaef1539d52dc35daeb28c1b322b947a00a3b7c15580212ba686'
+    aarch64: '59b1a498fa0cd809eb531a2cd8fce01c323dcedcc8abc87e87b6e3bfc044f455',
+     armv7l: '59b1a498fa0cd809eb531a2cd8fce01c323dcedcc8abc87e87b6e3bfc044f455',
+     x86_64: 'b7451454d0433777a136316614c9bc76cb7179757f74f5d2f4eacaf285758139'
   })
 
   depends_on 'at_spi2_core' # R
-  depends_on 'cdparanoia' # R
+  depends_on 'cairo' # R
+  depends_on 'cdparanoia'
+  depends_on 'fontconfig' # R
   depends_on 'freetype' # R
   depends_on 'gdk_pixbuf' # R
-  depends_on 'glibc' # R
   depends_on 'glib' # R
-  depends_on 'gtk2' # R
+  depends_on 'glibc' # R
+  depends_on 'gtk2'
```

Comment:
There is no way to build this without gtk3?

Reply:
Nope, the package uses gtk2.

Reply:
Sigh ...

https://littlesvr.ca/bugs/show_bug.cgi?id=59